samples/common/Kconfig

```diff
@@ -123,6 +123,7 @@ config GOLIOTH_SAMPLE_WIFI_SETTINGS
 	bool "Load SSID and PSK from settigs subsystem"
 	depends on GOLIOTH_SAMPLE_WIFI
 	depends on SETTINGS
+	default y if GOLIOTH_SAMPLE_PSK_SETTINGS
```

Comment:
I am not sure why PSK settings should enable (by default) WiFi settings. Those options sound like two independent features.

Reply:
It's more about user expectation, but I agree they aren't related. I'll enable this option in `samples/common/runtime_psk.conf` instead.